assembl/views/__init__.py

```diff
@@ -34,6 +34,7 @@
     User,
     Locale,
 )
+from assembl.models.discussion import Discussion
```

Comment:
please remove this import

Reply:
we should continue on flake8 (remove ignored files and fix the errors)

Reply:
@cedricmessiant agreed. I'm surprised flake8 didn't catch this!

Reply:
@ayyazdaniaryan it's because flake8 is currently only done mainly on models and graphql packages, see `exclude` in `.flake8` file.